src/extension/executors/github.ts

```diff
@@ -0,0 +1,35 @@
+import { NotebookCellExecution, TextDocument, authentication, window } from 'vscode'
+
+import type { Kernel } from '../kernel'
+import { NotebookCellOutputManager, updateCellMetadata } from '../cell'
+import { OutputType } from '../../constants'
+
+import { parseGitHubURL, getYamlFileContents } from './github/workflows'
+
+export async function github(
+  this: Kernel,
+  exec: NotebookCellExecution,
+  doc: TextDocument,
+  outputs: NotebookCellOutputManager,
+): Promise<boolean> {
+  try {
+    await authentication.getSession('github', ['repo'], { createIfNone: true })
```

Comment:
btw, `'github'` here should probably be a constant someplace.

Reply:
fixed